src/AzureStack/Admin/BackupAdmin/Backup.Admin.Tests/src/BackupTests.cs

```diff
@@ -57,7 +57,6 @@ public class BackupTests : BackupTestBase
                     {
                         var bName = ExtractName(backup.Name);
                         client.Backups.Get(ResourceGroupName, blName, bName);
-                        return;
```

Comment:
The point of this test was to just test getting a single backup.

Reply:
I see. But as PS test goes through each backup in the same case, I think we can align them so that session records can be reused.